tests/field/example-django.sh

```diff
@@ -40,6 +48,7 @@ done
 
 log 2 'Create demo project from scratch and push it'
 tox -e cookiecutter -- \
+    -o /tmp/painless-generated-project \
```

Comment:
Let's put the `-o` option into the [cookicutter target](https://github.com/painless-software/painless-continuous-delivery/blob/master/tox.ini#L49-L54) of `tox.ini`.

I believe, that would also make the [`changedir` option](https://github.com/painless-software/painless-continuous-delivery/blob/master/tox.ini#L52) unnecessary, so we could remove that.